guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -0,0 +1,28 @@
+[id="associating-partition-tables-with-disk-encryption_{context}"]
+= Associating partition tables with disk encryption
+
+{Project} contains partition tables that encrypt the disk of your host using Linux Unified Key Setup (LUKS) during host provisioning.
+Optionally, you can also bind the disk to a Tang server through Clevis for decryption during boot.
+You must associate the partition table with your operating system entry.
+Then, you assign the partition table to your host group or select it manually during provisioning.
+
+Encrypted disks on hosts protect data at rest.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning Setup* > *Operating Systems*.
+. Select your {client-os} entry.
+. On the *Partition Table* tab, associate the `{client-provisioning-template-type} default encrypted` to your operating system entry.
+For more information, see xref:creating-partition-tables_{context}[].
+. Create a host group that uses the `{client-provisioning-template-type} default encrypted` partition table.
+For more information, see {ManagingHostsDocURL}Creating_a_Host_Group_managing-hosts[Creating a host group] in _{ManagingHostsDocTitle}_.
++
+There are two options to decrypt the disk of your hosts during boot time:
++
+* You can add the host parameter `disk_enc_passphrase` as type *string* and your cleartext passphrase of the LUKS container as the value.
```

Comment:
```suggestion
* LUKS encryption: Add the host parameter `disk_enc_passphrase` as type *string* and your cleartext passphrase of the LUKS container as the value.
```
I think the list items would be easier to scan if you add a summary at the beginning of each list item.

Reply:
Applied; see my last commit. I've also reworded the second option. Is this what you had in mind @apinnick ?